README.md

```diff
@@ -0,0 +1,70 @@
+# OpenCompose
+
+[![Build Status](https://travis-ci.org/redhat-developer/opencompose.svg?branch=master)](https://travis-ci.org/redhat-developer/opencompose)
+
+The goal of OpenCompose is to make it easier to on-board developers to Kubernetes.
+It is a curated higher level abstraction for specific Kubernetes resources.
+Very simple idea, isn't it? Developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications on Kubernetes.
+The developer is generally concerned with the application that is being developed.
+
+We are at a very nascent stage of this project and we have listed some of our ideas as [issues](https://github.com/redhat-developer/opencompose/issues)
+and some of our ideas have taken shape in form of [examples](https://github.com/redhat-developer/opencompose/blob/master/examples/) 
+but we are definitely looking for suggestions and contributions from the Kubernetes community.
+Please send PRs to improve this project, file issues and RFEs against this repository.
+
+### Installation
+We don't have a release yet, but you can easily build OpenCompose tool from source.
+All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```sh
+go get -u https://github.com/redhat-developer/opencompose
+```
+
+### Usage
+```
+$ opencompose --help
+opencompose is a tool to transform OpenCompose files into Kubernetes (and OpenShift) definitions
```

Comment:
Sorry, I probably should have been clearer when requesting this: I was ok with just mentioning the `opencompose --help` command. It looks nice when embedded although it might pose a challenge to keep it up to date. But CI can make sure as we should do the same for examples. I will leave it up to you @kadel to decide ;)